hurricane/testing/drivers.py

```diff
@@ -24,44 +31,35 @@ def __init__(self):
                 try:
                     sock.bind(("127.0.0.1", port))
                 except OSError:
-                    raise Exception(f"Port {port} already in use.")
+                    raise BusyPortException(port)
```

Comment:
Same as with `HurricaneWSGIException`, please do not net the message in the constructor